mautic/mautic
# Feature/add rte for grapesjs TPROD-215

Adds the Rich Text Editor (RTE) CKEditor to the grapesjs plugin.

This brings the following features
- Format text inside a grapesjs textblock or button
- Add links inside a text block

![image](https://user-images.githubusercontent.com/13075514/130326868-4fb0c973-d2b1-412a-ab1b-a6581d2fd5be.png)


fixes: https://github.com/mautic/mautic/issues/10289
fixes: https://github.com/mautic/mautic/issues/10004

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Edit an email in HTML mode: check that you can use the features described above
3. Edit an email in MJML mode: check that you can use the features described above
4. Edit a landingpage: check that you can use the features described above
5. Check that there are no errors in the console
